src/index.js

```diff
@@ -83,7 +83,7 @@ const ClearAllBtn = () => {
   });
   let count = 0;
   const data = Array.from(localData).filter(i => i.completed === false);// eslint-disable-line
-  data.map(i => i.index = count += 1); // eslint-disable-line
+  data.map(i.index = count += 1); // eslint-disable-line
```

Comment:
:tada:

Reply:
if I don't disable the Eslint in this line my linters check will not pass.

Reply:
@descholar-ceo thanks for your excellent feedback I really appreciate it.
I wish you all the best! 💯